oaknational/Oak-Web-Application
# fix: 1224 remove green box

## Description

- List of changes
- removed flex and subject error components from unit and unitslug pages

## Issue(s)

Fixes #1224 

## How to test

1. Go to https://deploy-preview-1276--oak-web-application.netlify.app/
2. Click on 
example endpoints:
/beta/teachers/key-stages/ks1/subjects/art/units
/beta/teachers/key-stages/ks1/subjects/art/units/drawing-c92d
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

![Screenshot 2023-02-15 at 18 26 02](https://user-images.githubusercontent.com/91190841/219121700-0e8ce984-c3a6-4757-83e1-5fa48d4fe3e3.png)
![Screenshot 2023-02-15 at 18 25 01](https://user-images.githubusercontent.com/91190841/219121734-73e9e556-37e6-4916-af69-fc2c395ce4e1.png)


How it should now look:
{screenshots}

![Screenshot 2023-02-15 at 18 25 48](https://user-images.githubusercontent.com/91190841/219121786-dda4de5e-7d9e-402d-9408-ef3195e237d3.png)

![Screenshot 2023-02-15 at 18 25 31](https://user-images.githubusercontent.com/91190841/219121807-09b2cd19-4d38-4f7e-ac6e-ddecd9fa35ca.png)




## Checklist
